rfc/0003-indexmanagement.md

```diff
@@ -63,26 +85,41 @@ String state //eg. "online", "pending" or "deferred"/"building"
 String keyspace
 String namespace
 JsonArray indexKey //or equivalent Json representation
+String condition //empty string if the index has no WHERE clause
```

Comment:
this is the condition part of the index, when you've built it with a `whereClause` expression. The json returned by querying "system:indexes" can contain a `condition` field. If not present, the SDK object should have the empty string instead.